.travis.yml

```diff
@@ -370,12 +351,11 @@ jobs:
 
     - stage: test
       env:
-        - PROJECT=Firestore PLATFORM=iOS METHOD=xcodebuild QUICKSTART=firestore
+        - PROJECT=Firestore PLATFORM=iOS METHOD=xcodebuild
```

Comment:
You can strike this stanza altogether. Firestore does its xcodebuild testing on GHA already. All we need over here is Xcode 10.3, and even that hasn't turned up a single issue for Firestore so you could probably scratch that too.